data_model/src/events/data/filters.rs

```diff
@@ -1,192 +1,466 @@
-//! This module contains `EventFilter` and entities for filter
+//! This module contains filters for data events.
+//!
+//! (almost) Each event in [`super::events`], there's two corresponding types in this module:
+//! - `*EventMatcher` - matches one event kind (e.g. [`super::events::AccountEvent::Created`] with [`AccountEventMatcher::ByCreated`])
+//! - `*EventFilter` - struct combining an optional id matcher and an optional event matcher
+//!
+//! The ones not having a filter are [`super::events::ConfigurationEvent`] and [`super::events::ExecutorEvent`] (TODO: why?).
```

Comment:
why not?

Reply:
We didn't have them before the refactor and I don't know why really. Maybe the one who added them forgot to include in the `DataEventFilter` (or whatever it was called before), but maybe there's a deeper reason I can't see...

Reply:
remove the comment